metals/src/main/scala/scala/meta/internal/metals/ScalacDiagnostic.scala

```diff
@@ -30,4 +30,22 @@ object ScalacDiagnostic {
         case _ => None
       }
   }
+
+  object SymbolNotFoundWithImportSuggestions {
```

Comment:
I also wonder if we could get this from the compiler itself and maybe use completions?